service/src/main/java/bio/terra/catalog/api/DatasetApiController.java

```diff
@@ -88,4 +88,10 @@ public ResponseEntity<DatasetPreviewTable> getDatasetPreviewTable(UUID id, Strin
         .cacheControl(CacheControl.noStore())
         .body(datasetService.getDatasetPreview(getUser(), new DatasetId(id), tableName));
   }
+
+  @Override
+  public ResponseEntity<Void> exportDataset(UUID datasetId, UUID workspaceId) {
```

Comment:
Do you think it would be worth filing a tech debt ticket to change the other param names to be `datasetId` as well?

Reply:
Done.